runtime/compiler/optimizer/DataAccessAccelerator.hpp

```diff
@@ -187,7 +187,7 @@ class TR_DataAccessAccelerator : public TR::Optimization
     *  \detail
     *     Unlike constant precision DAA call inlining which can be done in-place without introducing extra blocks,
     *     each variable precision call node has to be bloated into multiple
-    *     basic blocks to form a precision diamon. This disrupts the CFG and invalidates block iterator and
+    *     basic blocks to form a precision diamon.. This disrupts the CFG and invalidates block iterator and
```

Comment:
Why the extra `.`?

Reply:
I'll update all of these once you get an answer to your question below. (They're locally fixed)